pallets/gear/src/benchmarking/mod.rs

```diff
@@ -1656,6 +1673,15 @@ benchmarks! {
         verify_process(res.unwrap());
     }
 
+    mem_grow {
+        let r in 0 .. API_BENCHMARK_BATCHES;
+        let mut mem = MemoryWrap::new(DefaultExecutorMemory::new(1, None).unwrap());
```

Comment:
Need to pin memory type. So make in core-backend lib.rs:
```rust
pub use DefaultExecutorMemory;
```
And reuse it here in order to be sure that we use the same memory as in sandbox.